docs/source/openapi.json

```diff
@@ -1105,13 +1105,7 @@
         "type": "object",
         "required": [
           "nan_count",
-          "nan_proportion",
-          "min",
-          "max",
-          "mean",
-          "median",
-          "std",
-          "histogram"
+          "nan_proportion"
```

Comment:
actually not sure about this change, all keys should be in response, just some of them might be null. should i revert?

Reply:
I just checked and if the fields are always in the response (regardless of whether they're null or not) then they can can be marked as "required"

So you can revert this change imo